srcalc/src/test/java/gov/va/med/srcalc/domain/calculation/SampleCalculations.java

```diff
@@ -29,10 +30,26 @@ public static VistaPerson radiologistPerson()
                 Optional.of("Physicians (M.D. and D.O.)"));
     }
 
+    /**
+     * Returns a sample patient with minimal attributes populated.
+     */
     public static Patient dummyPatient(final int dfn)
     {
-        final Patient patient = new Patient(dfn, "Zach Smith", "M", 40);
-        patient.setBmi(new RetrievedValue(20.0, new Date(), ""));
+        return new Patient(dfn, "Zach Smith", Gender.Male, 40);
+    }
+    
+    /**
+     * Returns a sample patient with vitals (height, weight, bmi) populated.
+     */
+    public static Patient dummyPatientWithVitals(final int dfn)
+    {
+        final Patient patient = dummyPatient(dfn);
+        final DateTime vitalsMeasureDate = new DateTime(2014, 5, 12, 13, 21);
+        patient.setBmi(new RetrievedValue(20.0, vitalsMeasureDate.toDate(), ""));
```

Comment:
I think it would also be good to refactor the RetrievedValue class to a DateTime as well.